ui/src/app/machines/views/MachineList/OwnerColumn/OwnerColumn.js

```diff
@@ -0,0 +1,33 @@
+import { useSelector } from "react-redux";
+import React from "react";
+import PropTypes from "prop-types";
+
+import { machine as machineSelectors } from "app/base/selectors";
+
+const OwnerColumn = ({ systemId }) => {
+  const machine = useSelector(state =>
+    machineSelectors.getBySystemId(state, systemId)
+  );
+
+  const owner = machine.owner ? machine.owner : "-";
+  const tags = machine.tags ? machine.tags.join(", ") : "";
+
+  return (
+    <>
```

Comment:
I would take `p-double-row` off the `th` and put it here, personally. It might seem unnecessarily nested but I've found tweaking padding/margin on `th`s directly can sometimes mess with the Vanilla defaults and ruin everything. D: